danielMunteanu/src/main/java/danielCollectionTask/AdminUser.java

```diff
@@ -0,0 +1,23 @@
+package danielCollectionTask;
+
+import java.util.Map;
+
+
+public class AdminUser extends AbstractUser {
+
+    public AdminUser(String username, Map<String, Address> addresses) {
+        super(username, addresses);
+    }
+
+    public AdminUser(String username) {
+        super(username);
+    }
+
+    public void approveOrder(Order order) {
+
+    }
+
+    public void rejectOrder(Order order) {
+
```

Comment:
don't let them be empty. Put a sout here and make sure you call these methods at least once

Reply:
done